core/DataAccess/Model.php

```diff
@@ -417,7 +422,7 @@ public function deleteOlderArchives(Parameters $params, $name, $tsArchived, $idA
         $numericTable = ArchiveTableCreator::getNumericTable($dateStart);
         $blobTable = ArchiveTableCreator::getBlobTable($dateStart);
 
-        $sql = "SELECT idarchive FROM `$numericTable` WHERE idsite = ? AND date1 = ? AND date2 = ? AND period = ? AND name = ? AND ts_archived < ? AND idarchive < ?";
+        $sql = "SELECT idarchive FROM `$numericTable` WHERE idsite = ? AND date1 = ? AND date2 = ? AND period = ? AND name = ? AND ts_archived <= ? AND idarchive < ?";
```

Comment:
This is a small fix, which caused multiple archives to exist for the same parameters when they were created in the same second. (See update test)